src/browser/index.ts

```diff
@@ -1,4 +1,9 @@
-function noop() { return (target: Object, propertyName: string) => {}; }
+function noop() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We should avoid as much as possible to use `eslint-disable-next-line` comments.
What does the function ? It seems it does nothing, maybe we don't need it anymore.

Reply:
Agreed, there are minimal occurrences in the code with these exceptions. But again, I didn't want to rewrite the codebase just because someone's rule has different opinion :-)

Reply:
Right, maybe we'll need to look, what need to be done to remove this comment before merging.

Reply:
Any ideas...? :-) It's not a bad thing to very sparingly use it (there is a reason why eslint allows disables like this ) ... If use them too often, you should rethink your ruleset. I mean e.g. in this case, I can see why it's written like this to keep the code consistent about the implementation ... even though vars are not used. If you remove them, you loose a bit of context and about what arguments are accepted. So there is always some trade-off.